includes/snmp.inc.php

```diff
@@ -386,7 +386,9 @@ function snmp_walk($device, $oid, $options = null, $mib = null, $mibdir = null)
     $data = str_replace('"', '', $data);
     $data = str_replace('End of MIB', '', $data);
 
-    if (is_string($data) && (preg_match('/No Such (Object|Instance)/i', $data))) {
+
+    if (is_string($data) && (preg_match('/No Such (Object|Instance)/i', $data) || preg_match('/Wrong Type(.*)should be/',$data))) {
```

Comment:
could you remove empty line before this line
and change `,$data`  to `, $data`


>

Reply:
Thanks.  done.